// currency-system/v14-ExampleBot/src/commands/bulkremovemoney.js
const CurrencySystem = require("currency-system");
const cs = new CurrencySystem;
exports.run = async (client, message, args) => {
    let wheretoPutMoney = args.get('where_to_put_money');
    if (wheretoPutMoney) wheretoPutMoney = 'bank';
    else wheretoPutMoney = 'wallet';
    let amount = args.get('amount')
    let money = parseInt(amount);
    let result = await cs.removeMoneyFromAllUsers({
        guild: message.guild.id,
        amount: money,
        wheretoPutMoney: wheretoPutMoney
    });
    if (result.error) {
        if (result.type === 'negative-money') return message.reply("You cant remove negitive money");
        else return message.reply('No User\'s found');
    } else message.reply(`Successfully removed $${money} from ${result.rawData.length} people!, ( in ${wheretoPutMoney} )`)
}

exports.help = {
    name: "removemoneyfromallusers",
    data: {
        name: 'removemoneyfromallusers',
        description: "A way to remove the amount  of money in everyone's bank or wallet.",
        options: [{
                name: 'amount',
                type: 4,
                description: 'Amount of money you want to remove.',
                required: true,
            },
            {
                name: 'where_to_put_money',
                type: 5,
                description: 'TRUE means bank, FALSE means wallet.',
                required: true,
            }
        ]
    }
}
