// currency-system/v14-ExampleBot/src/commands/deposite.js
    const CurrencySystem = require("currency-system");
    const cs = new CurrencySystem;
    exports.run = async (client, message, args) => {

        let money = args[0].value
        if (!money) return message.reply("Enter the amount you want to deposite.");

        let result = await cs.deposite({
            user: message.user,
            guild: message.guild,
            amount: money
        });
        if (result.error) {
            if (result.type === 'money') return message.reply("Specify an amount to deposite");
            if (result.type === 'negative-money') return message.reply("You can't deposite negative money");
            if (result.type === 'low-money') return message.reply("You don't have that much money in wallet.");
            if (result.type === 'no-money') return message.reply("You don't have any money to deposite");
            if (result.type === 'bank-full') return message.reply("Your bank is full. It has reached it's limit.");
        } else {
            if (result.type === 'all-success') return message.reply("You have deposited all your money to your bank" + `\nNow you've $${result.rawData.wallet} In your wallet and $${result.rawData.bank} in your bank.`);
            if (result.type === 'success') return message.reply(`You have deposited $${result.amount} money to your bank.\nNow you've $${result.rawData.wallet} In your wallet and $${result.rawData.bank} in your bank.`);
        };
    }

    exports.help = {
        name: "deposite",
        data: {
            name: 'deposite',
            description: "A way to get money in of the bank.",
            options: [{
                name: 'amount',
                type: 4,
                description: 'Amount of money you want to deposite.',
                required: true,
            }]
        }
    }

    exports.conf = {
        aliases: ["dep"],
        cooldown: 5
    }
