// currency-system/v14-ExampleBot/src/commands/sell.js
const Discord = require('discord.js')
const CurrencySystem = require("currency-system");
const cs = new CurrencySystem;
exports.run = async (client, message, args) => {
    if (!args[0].value) return message.reply('Which item to remove?')
    let result = await cs.removeUserItem({
        user: message.user,
        guild: message.guild,
        item: parseInt(args[0].value)
    });
    if (result.error) {
        if (result.type == 'Invalid-Item-Number') return message.reply('There was a error, Please enter item number to remove.!')
        if (result.type == 'Unknown-Item') return message.reply('There was a error, The Item Does not exist!')
    } else message.reply('Done! Successfully sold the `' + result.inventory.name + '` for free! You now have ' + result.inventory.amount + ' of those items left!')



}

exports.help = {
    name: "sell",
    data: {
        name: 'sell',
        description: "A way to sell item",
        options: [{
            name: 'item',
            type: 4,
            description: 'Item Number from Inventory',
            required: true,
        }]
    }
};

exports.conf = {
    aliases: [],
    cooldown: 5 // This number is a seconds, not a milliseconds.
    // 1 = 1 seconds.
}
